getting-started/developers-guide.md

```diff
@@ -2,65 +2,31 @@
-
-    $ git clone https://github.com/emscripten-core/emsdk.git
-    $ cd emsdk
-    $ ./emsdk install --build=Release sdk-incoming-64bit binaryen-master-64bit
-    $ ./emsdk activate --build=Release sdk-incoming-64bit binaryen-master-64bit
-
-## Environment setup after installation (non Windows 7 installs)
-After installing or compiling the SDK, the installation is complete. To enter an Emscripten compiler environment in the current command line prompt after downloading a precompiled toolchain or building your own, type
-
-    $ source ./emsdk_env.sh --build=Release
-
-This command adds relevant environment variables and directory entries to PATH to set up the current terminal for easy access to the compiler tools.
-
-## Compile and run a simple program
-We now have a full toolchain we can use to compile a simple program to WebAssembly. There are a few remaining caveats, however:
-
-- If we want Emscripten to generate an HTML page that runs our program, in addition to the Wasm binary and JavaScript wrapper, we have to specify an output filename with a `.html` extension.
-- Finally, to actually run the program, we cannot simply open the HTML file in a web browser because cross-origin requests are not supported for the `file` protocol scheme. We have to actually serve the output files over HTTP.
-
-The commands below will create a simple "hello world" program and compile it. The compilation step is highlighted in bold.
-
-<pre>
-$ mkdir hello
-$ cd hello
-$ cat << EOF > hello.c
-#include &lt;stdio.h&gt;
-int main(int argc, char ** argv) {
-  printf("Hello, world!\n");
-}
-EOF
-$ <b>emcc hello.c -o hello.html</b>
-</pre>
-
-To serve the compiled files over HTTP, we can use the `emrun` web server provided with the Emscripten SDK:
-
-    $ emrun --no_browser --port 8080 .
-
-Once the HTTP server is running, you can <a href="http://localhost:8080/hello.html" target="_blank">open it in your browser</a>. If you see "Hello, world!" printed to the Emscripten console, then congratulations! You've successfully compiled to WebAssembly!
+# I want to...
+
+- Compile a WebAssembly module from...
+  - C/C++
+    - [starting from scratch](https://developer.mozilla.org/en-US/docs/WebAssembly/C_to_wasm)
+    - [library that I want to port to the Web](https://developer.mozilla.org/en-US/docs/WebAssembly/existing_C_to_wasm)
+  - [Rust](https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm)
+  - [AssemblyScript (a TypeScript-like syntax)](https://docs.assemblyscript.org/)
+  - [C#](https://docs.microsoft.com/en-us/aspnet/core/blazor/get-started?view=aspnetcore-3.1&tabs=visual-studio)
+  - [F#](https://fsbolero.io/docs/)
+  - Go
+    - [with full language support](https://github.com/golang/go/wiki/WebAssembly#getting-started)
+    - [targeting minimal size](https://tinygo.org/webassembly/webassembly/)
+  - [Kotlin](https://kotlinlang.org/docs/reference/native-overview.html)
```

Comment:
Is there a more wasm-specific link we can point to here?

Reply:
I used the best I could find (it seems Kotlin/Native is sharing same official docs across supported targets), but if there is, I'm happy to change.

Reply:
(I found some 3rd-party blog posts but not feeling very comfortable linking out to them)

Reply:
OK, the link you have is pretty good for now. We can always update later if we find a better one.